python/django_bolt/serialization.py

```
@@ -273,9 +274,149 @@ def _response_validation_is_required(annotation: Any) -> bool:
+        return [item.dump() for item in items]
+    if sync_safe:
+        instances = [serializer_cls.from_model(item) for item in items]
+    else:
+        try:
+            instances = [serializer_cls.from_model(item) for item in items]
+        except UnloadedRelationError:
+            instances = list(await asyncio.gather(*(serializer_cls.afrom_model(item) for item in items)))
+    return serializer_cls.dump_many(instances)
+
+
+def _serializer_from_model_sync_safe(serializer_cls: Any) -> bool:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `serializer_cls`

(ANN401)